src/NLog.Web.AspNetCore/README.md

```diff
@@ -0,0 +1,35 @@
+# NLog.Web.AspNetCore
+
+- [Logging provider](https://learn.microsoft.com/en-us/dotnet/core/extensions/logging-providers) for ASP.NET Core platform via extension methods for IHostBuilder, and IWebHostBuilder.
```

Comment:
Instead of having this as feature-bullet, then this should be the introduction (below the headline)

```
Integrates NLog as [Logging provider](https://learn.microsoft.com/en-us/dotnet/core/extensions/logging-providers) for the ASP.NET Core platform by just calling `UseNLog()` on the host-builder.
```